pages/segment_trip_time.py

```diff
@@ -0,0 +1,326 @@
+from dash import dcc, html, Input, Output, State, callback, register_page, dash_table
+import dash_bootstrap_components as dbc
+import dash_leaflet as dl
```

Comment:
I agree that the mapbox maps that we have are currently unintuitive. They try to fit the plot interface with pan and zoom, but it always ends up centering me when I am trying to zoom around and then I have to go zoom out and find where I was again.

Reply:
@achasmita, we should probably replace the other mapbox maps with leaflet as well.